ddtrace/tracer/spancontext.go

```diff
@@ -111,6 +111,8 @@ type spanContext struct {
 	baggage    map[string]string
 	hasBaggage uint32 // atomic int for quick checking presence of baggage. 0 indicates no baggage, otherwise baggage exists.
 	origin     string // e.g. "synthetics"
+
+	spanLinks []ddtrace.SpanLink // links to other conflicting traces
```

Comment:
Why are they "conflicting"? I thought that span links were just links to other relevant spans.

Reply:
They are relevant spans that have conflicting trace information from the first trace that we have extracted. Since these traces were all propagated together from a distributed system, this is how we are choosing to show their relationship. [RFC](https://docs.google.com/document/d/1xacBonCyuVk95D-L1STXGqdLtsOuf0_3jG-138uXHnQ/edit?pli=1&tab=t.0#bookmark=id.uwnmpxy1znny)